lib/server/ForwardClientRequest.js

```diff
@@ -1,31 +1,37 @@
 "use strict";
 
-var Promise = require("bluebird");
```

Comment:
i think github might be confused. the move looks correct when i go to the commit where this happened: https://github.com/bloomberg/brisket/pull/287/commits/fb9b621395f1a2894155675b4a57e6abfd4bf7f2#diff-d59d7f2ed9ffc5b6bd4fdd13004df2a0L1